provider-ui/pages/auth/login.js

```diff
@@ -70,7 +70,7 @@ const AlertUnauthenticatedSession = ({ classes }) => {
       >
         <DialogTitle id="alert-dialog-title" className={classes.title}>
           <span className={classes.icon}>
-            <InfoIcon color="#2196f3"/>
+            <WarningIcon color="#F0D053"/>
```

Comment:
Import this icon, otherwise, it won't work.

Reply:
ok would provide a ss :) 
Oh how did i miss i will import it right away

Reply:
It did work out as I had imported it earlier the reason for it to not not be present in commit was coz I was messing with the signing off , 
so I was trying to do little changes to commit and sign off :) but everything is good now .